backend.rb

```diff
@@ -25,13 +26,16 @@ class Backend < Sinatra::Base
     }
   }
 
+  # Handle POST requests to '/pay'
   post '/pay' do
+    # Make key-value pairs from the JSON body available in the 'params' hash
```

Comment:
`Make key-value pairs from the JSON body available in the 'params' hash`  ~> `Insert the key-value pairs from the JSON body into the 'params' hash` ?